### DISCUSSION TITLE
Use chips in ui.input

### DISCUSSION THREAD

**Discussion by Stefan2016** [zauberzeug/nicegui#1649] [Ideas / Feature Requests]
I would love to have an input field where the user can input words and by pressing enter a chip is created, which itself can be removed/ deleted. 

In an older Quasar framework version you can see this here: https://v0-17.quasar-framework.org/components/chips-input.html
![chips](https://github.com/zauberzeug/nicegui/assets/17859809/96921bb2-b3ad-4853-8d6f-0c389d90f1ac)

Source code link: [https://github.com/quasarframework/quasar-play/blob/master/src/pages/showcase/forms/chips-input.vue](https://github.com/quasarframework/quasar-play/blob/master/src/pages/showcase/forms/chips-input.vue)

**Comment by BaccanoMob** [comment]
Apparently its possible now

```python
    ui.select(
        [],
        label="Add something",
        new_value_mode="add-unique",
        multiple=True,
    ).props("use-chips")
```

Also this will make the dropdown (and its icon) disappear, making it input only

```python
    ui.select(
        [],
        label="Add something",
        new_value_mode="add-unique",
        multiple=True,
    ).props("use-chips hide-dropdown-icon popup-content-class='hidden'")
```

**Reply by falkoschindler** [reply]
As mentioned in https://github.com/zauberzeug/nicegui/pull/4940#issue-3203522773, this solution is flawed:
1. Type "abc"+Enter --> (abc)
2. Type "def"+Enter --> (abc)(def)
3. Type "a"+Enter --> (def)

The expected result would be (abc)(def)(a), but typing "a" selects "abc" in the hidden dropdown, causing (abc) to disappear when pressing Enter.

I'll re-open this feature request.

**Comment by falkoschindler** [comment]
This feature just got released in version 2.22.0.
